src/components/organisms/ChatSidebar/components/VenueChat/VenueChat.tsx

```diff
@@ -31,8 +34,7 @@ export const _VenueChat: React.FC<VenueChatProps> = ({ venue }) => {
         messages={messagesToDisplay}
         sendMessage={sendMessage}
         sendThreadReply={sendThreadReply}
-        deleteMessage={deleteMessage}
-        venue={venue}
+        deleteMessage={canDeleteMessages ? deleteMessage : undefined}
```

Comment:
Here we cannot simplify to `&&` expression because type of `canDeleteMessages` is boolean and the resulting type of `canDeleteMessages && deleteMessage` is `Func | false` and TS complains about it. However, I simplified it in all other places across the PR.